### DISCUSSION TITLE
Alternative Menu

### DISCUSSION THREAD

**Discussion by JStBout** [Shopify/dawn#3553] [Q&A]
Hi Guys I am relatively new to my Shopify code section "rookie" and I would like someone who can help explain the steps that I need to follow to add an alternative menu to my Shopify store. I want to do this because I want a different menu to appear on only some of my pages. for example, I would like a menu on my products page with tops-bottoms-outerwear- etc.. At the moment my default menu appears on every page that you go into. Hopefully, someone can interpret what I want. I would like the default header on my home page to remain the same as it is.

I have looked at other posts similar to mine and they have not been very helpful and very hard to follow. If someone could step it out in very basic instructions that would be great

**Comment by joshistoast** [comment]
All header snippets refer to the section setting's `menu`, so add a more flexible source of truth maybe called `current_menu` or something. And just to make it even better add a string metafield on pages for menu handles. Then in your theme code adjust so that if a page has a menu handle metafield use that instead of the section setting.
